package.json

```diff
@@ -150,11 +152,21 @@
                                 "description": "Server IP/Domain address.",
                                 "default": "127.0.0.1"
                             },
+                            "protocol": {
+                                "type": "string",
+                                "description": "Server protocol (http or https).",
+                                "default": "http"
+                            },
                             "port": {
                                 "type": "number",
                                 "description": "The WsWorkunits port.",
                                 "default": 8010
                             },
+                            "rejectUnauthorized": {
+                                "type": "boolean",
+                                "description": "Reject unauthorized calls e.g. SSL certificate errors.",
+                                "default": true
```

Comment:
The default value for rejectUnauthorized in the request object is true so i just maintained the default behaviour